recipes/neurokit2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "NeuroKit2" %}
+{% set version = "0.0.42" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2881a7fb8a3c4a2e92360f4068b865b80957a0ed2f53454e91523a6a3127f59
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```

Comment:
it looks like you'll need `pytest-runner` as well since it's listed in `setup_requires` in the `setup.py`.

Reply:
Good catch, thanks! Let's see if it works now.